dapps/react-dapp-v2/src/chains/hedera.ts

```diff
@@ -0,0 +1,58 @@
+import { JsonRpcRequest } from "@walletconnect/jsonrpc-utils";
+
+import { HEDERA_LOGO_URL } from "../constants";
+import {
+  NamespaceMetadata,
+  ChainMetadata,
+  ChainRequestRender,
+  ChainsMap,
+} from "../helpers";
+
+export const HederaMetadata: NamespaceMetadata = {
+  testnet: {
+    logo: HEDERA_LOGO_URL,
+    rgb: "118, 90, 234",
+  },
+};
+
+export const HederaChainData: ChainsMap = {
+  testnet: {
+    name: "Hedera Testnet",
+    id: "hedera:testnet",
+    /**
+     * Hedera integration utilizes an sdk instead of a JSON-RPC relay:
+     * {@link https://specs.walletconnect.com/2.0/blockchain-rpc/hedera-rpc}
+     */
+    rpc: [],
+    slip44: 3030,
+    testnet: true,
+  },
+};
+
+export function getChainMetadata(chainId: string): ChainMetadata {
+  const reference = chainId.split(":")[1];
+  const metadata = HederaMetadata[reference];
+  if (typeof metadata === "undefined") {
+    throw new Error(`No chain metadata found for chainId: ${chainId}`);
+  }
+  return metadata;
+}
+
+export function getChainRequestRender(
+  request: JsonRpcRequest
+): ChainRequestRender[] {
+  let params = [{ label: "Method", value: request.method }];
+
+  switch (request.method) {
+    default:
```

Comment:
A recent PR was graceful enough to factor out the "default-only" `switch` cases where a switch wasn't actually needed, could you do the same here if it's also redundant in this case? E.g. https://github.com/WalletConnect/web-examples/blob/main/dapps/react-dapp-v2/src/chains/near.ts#L36